src/storage/memory_store.ts

```diff
@@ -31,6 +31,36 @@ const DEFAULT_WALLET_DATA = {
   gapLimit: GAP_LIMIT,
 };
 
+/**
+ * Get the ordering key for a transaction.
+ * This will be used to sort the transactions by timestamp.
+ *
+ * @param {IHistoryTx} tx The transaction
+ * @returns {string} The ordering key for the transaction
```

Comment:
Since it's not intuitive what a unsigned it to hexa would look like, an `@example` tag would be welcome here, with a sample of the return value.

Reply:
Added examples